lib/testing-helpcenter/help_center_version_helper.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module TestingHelpCentreOnlyoffice
+  # Help center helper for getting current hash
+  class HelpCenterVersionHelper
+    def self.fetch_uri
```

Comment:
Maybe both methods should be not `self.something`, but place them both in config class, since they are related to config

Also in current variant I dont think they will work if `config` is not initialized  in `spec_helper` - so if this code will not work without rspec - this code should be in `spec` folder, but I dont think it's a proper place for it

Reply:
code is working